pkg/docker/overview.js

```diff
@@ -179,6 +179,7 @@ define([
             var added = false;
             if (!tr.length) {
                 var button = $('<button class="btn btn-default btn-control fa fa-play">').
+                    attr("title", "run image").
```

Comment:
Thanks. Could you capitalize the 'R' and also wrap this in a ```_()``` call like this: ```_("Run image")``` ... which makes this string translatable.